### PR TITLE
fix(ripple): Double trigger on mobile

### PR DESCRIPTION
**Pull request by aluminick**

### CODE REVIEW

File: packages/ripple/index.tsx [hunk 1]
```diff
@@ -258,10 +258,7 @@ export function withRipple <
 
   activateRipple = (e: ActivateEventTypes<Surface>) => {
     // https://reactjs.org/docs/events.html#event-pooling
-    e.persist();
```
[the remark this answers is not in the document]
Comment (aluminick) [reply packages/ripple/index.tsx]: ok sir

[the remark this answers is not in the document]
Comment (aluminick) [reply packages/ripple/index.tsx]: > You still need to keep the e.persist() here.

Ok Sir

[the remark this answers is not in the document]
Comment (aluminick) [reply packages/ripple/index.tsx]: I think the RAF inside the foundation is the one mocked in the unit testing and not the RAF inside the `activateRipple`. I don't think it makes sense to wrap a RAF(foundation.activate) with a RAF but I maybe wrong.

[the remark this answers is not in the document]
Comment (aluminick) [reply packages/ripple/index.tsx]: > I think the RAF inside the foundation is the one mocked in the unit testing and not the RAF inside the `activateRipple`. I don't think it makes sense to test a RAF(foundation.activate) wrapping a RAF, but I maybe wrong.

[the remark this answers is not in the document]
Comment (aluminick) [reply packages/ripple/index.tsx]: I think the RAF inside the foundation is the one mocked in the unit testing and not the RAF inside the activateRipple. I don't think it makes sense to test a RAF(foundation.activate) wrapping a RAF, but I maybe wrong.

[the remark this answers is not in the document]
Comment (aluminick) [reply packages/ripple/index.tsx]: I think the RAF inside the foundation is the one mocked in the unit testing and not the RAF inside the activateRipple. I don't think it makes sense to test a RAF wrapping a RAF. But I maybe wrong please tell me.

[the remark this answers is not in the document]
Comment (moog16) [reply packages/ripple/index.tsx]: I believe you're correct, but is this actually fixing the double ripple on mobile?

**Comment by moog16** [comment]
Ripple doesn't look better though. 
[doubleripple.mov.tar.gz](https://github.com/material-components/material-components-web-react/files/2784293/doubleripple.mov.tar.gz)

**Comment by moog16** [comment]
sounds good - please ping me in the PR when its ready again.

**Comment by aluminick** [comment]
Opps sorry wrong push.

**Comment by moog16** [comment]
@aluminick I updated to rc0.10.0. Please merge in rc0.10.0 to your branch and fix any conflicts

**Comment by moog16** [comment]
closing this as we had a new PR that was just merged

**Pull request closed without merging**